### PR TITLE
Figure

### PR DESCRIPTION
**Pull request by FaGru**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,9 +14,31 @@ <h1>Fabian Gruber</h1>
...
       </nav>
     </header>
+    <figure>
+      <img src="https://source.unsplash.com/random/600x300" alt="">
+      <figcaption>nice picture</figcaption>
+    </figure>
+
     <section id="contact">
       <h2>Contact</h2>
+      <u>Phone:
```
Comment (J-Massmann) [inline index.html:24]: I am afraid u typed "u" instead of "ul" 👀

File: index.html [hunk 2]
```diff
@@ -14,9 +14,31 @@ <h1>Fabian Gruber</h1>
...
+        <li>privat: 12345/67890</li>
+        <li>mobile: 09876/54321</li>
+      </u>
+      <p></p>
```
Comment (J-Massmann) [inline index.html:28]: Maybe leave out the <p> tag?

**Review by MaggieSchuetz — COMMENTED** [review]
Nice work! May I ask what the <p></p> in like 28 is for? Asking because I actually don't know if I missed something in my own code and should have put that there.

**Comment by J-Massmann** [comment]
All in all your code looks good, it just needs some tiny changes 👍

**Review by J-Massmann — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (FaGru) [reply index.html:28]: i wanted a empty line... because of this there is the `<p>` without text in it

**Pull request merged**